.travis/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Can you elaborate why you removed the `echo_run` function? I have found it to be helpful when debugging build-time issues in Travis.